app/actions/views/channel.js

```diff
@@ -545,31 +552,39 @@ export function setChannelDisplayName(displayName) {
 }
 
 // Returns true if there are more posts to load
-export function increasePostVisibility(channelId, focusedPostId, direction = ListTypes.VISIBILITY_SCROLL_UP) {
+export function increasePostVisibility(channelId, postId, direction = ListTypes.VISIBILITY_SCROLL_UP) {
     return async (dispatch, getState) => {
         const state = getState();
-        const {loadingPosts, postVisibility} = state.views.channel;
-        const currentPostVisibility = postVisibility[channelId] || 0;
+        const {loadingPosts} = state.views.channel;
+
+        // const {postVisibility} = state.views.channel;
```

Comment:
did you forget to remove the comment?

Reply:
No, this is just only used below by the commented out bi-directional scrolling logic.